app/src/main/java/com/unithon/ott/presentation/ui/question/QuestionWriteViewModel.kt

```diff
@@ -0,0 +1,14 @@
+package com.unithon.ott.presentation.ui.question
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class QuestionWriteViewModel : ViewModel() {
+
+    var nickname : MutableLiveData<String> = MutableLiveData()
```

Comment:
로컬에서 가져올거면 `mutableLiveData` 말고 그냥 `String`으로 선언하고 레포지토리에서 값 get 해도 될거 같아요
```suggestion
    val nickname : String = repository.getNickname()
```